LoopKit/GlucoseKit/SyncGlucoseObject.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  SyncGlucoseObject.swift
+//  LoopKit
+//
+//  Created by Darin Krauss on 10/13/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+
+public struct SyncGlucoseObject: Codable, Equatable {
```

Comment:
Why do we need a new type for watch?  Seems like watch should use the same types that Loop uses. Looks very similar to StoredGlucoseSample

Reply:
Yes, it is very similar to StoredGlucoseSample, but I chose to use a separate type for 2 reasons:
1. StoredGlucoseSample would require a manually implemented Codable since it uses an `HKQuantity` (which cannot directly be Codable).
2. For symmetry with CarbStore.
If you want me to use StoredGlucoseSample I can implement the custom Codable implementation. Let me know.

Reply:
RE no. 1: I don't think this is a good reason to create a new type. We interact with the types all the time in code, and have to reason about them all the time; the serialization is more of a one time thing.

RE no. 2: I'm not sure I follow what symmetry you're referring to.

Reply:
Pending results of other thread regarding WatchHistoricalGlucose I will remove SyncGlucoseObject and use custom Codable implementation of StoredGlucoseSample.

Reply:
Removed SyncGlucoseObject and replaced with StoredGlucoseSample.